ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ip-nat.md

```diff
@@ -112,41 +112,49 @@ NAT profile VRF is: TEST
 
 | Setting | Value |
```

Comment:
IMO this whole section is very hard to read. I would rather have a single table with NAT pools. One line per pool, containing a column per relevant thing. First and last ports should just be `<first_port>-<last_port>` and multiple should be joined with comma.

Reply:
I will create a new table.

Reply:
Done!